src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/IaasCmdletInfo/SetAzureDeploymentCmdletInfo.cs

```diff
@@ -38,12 +39,13 @@ public static SetAzureDeploymentCmdletInfo SetAzureDeploymentStatusCmdletInfo(st
         }
 
 
-        public static SetAzureDeploymentCmdletInfo SetAzureDeploymentConfigCmdletInfo(string serviceName, string slot, string configPath)
+        public static SetAzureDeploymentCmdletInfo SetAzureDeploymentConfigCmdletInfo(string serviceName, string slot, string configPath, ExtensionConfigurationInput extConfig = null)
         {
             SetAzureDeploymentCmdletInfo result = new SetAzureDeploymentCmdletInfo(serviceName, slot);
 
             result.cmdletParams.Add(new CmdletParam("Config"));
             result.cmdletParams.Add(new CmdletParam("Configuration", configPath));
+            result.cmdletParams.Add(new CmdletParam("ExtensionConfiguration", extConfig));
```

Comment:
Are Set-AzureDeployment with "-ExtensionConfiguration $null" parameter and Set-AzureDeployment without -ExtensionConfiguration paramter the same?
If they are the same, this line should be added when 'extConfig' is not null.
If they are not the same, we need two separate methods, with $null extConfig and without extConfig.